src/reportWebVitals.js

```diff
@@ -0,0 +1,15 @@
+const reportWebVitals = (onPerfEntry) => {
+  if (onPerfEntry && onPerfEntry instanceof Function) {
+    import('web-vitals').then(({
+      getCLS, getFID, getFCP, getLCP, getTTFB,
+    }) => {
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
+    });
+  }
+};
+
+export default reportWebVitals;
```

Comment:
- [ **OPTIONAL** ] To make your project looks clean and to have the small size of storage, I think it would be better if you remove/delete all the files which are not related to your project.